service/status-report/main.go

```diff
@@ -206,3 +226,21 @@ func setPlatformInformation(ctx context.Context, cl client.Client, status interf
 	}
 	status.SetPlatformVersion(platformVersion)
 }
+
+func extractNoobaaResouces(noobaaData []byte) (string, string, error) {
+	data := map[string]string{}
+	err := json.Unmarshal(noobaaData, &data)
+	if err != nil {
+		return "", "", fmt.Errorf("failed to unmarshall data: %v", err)
+	}
+	noobaaAuthToken, ok := data["auth_token"]
+	if !ok {
+		return "", "", fmt.Errorf("noobaa auth token not found")
+	}
+	noobaaMgmtAddress, ok := data["mgmt_addr"]
+	if !ok {
+		return "", "", fmt.Errorf("noobaa mgmt address not found")
+	}
```

Comment:
Isn't this a redundant check? We already check if the values are empty on the ocs-operator side

Reply:
I think it is ok, since I am extracting and checking at the same point